Source/SlimDetours/Memory.c

```diff
@@ -130,6 +130,16 @@ detour_memory_free(
     return RtlFreeHeap(_detour_memory_heap, 0, BaseAddress);
 }
 
+VOID
+detour_memory_uninitialize(VOID)
+{
+    if (_detour_memory_heap != NULL)

```

Comment:
```suggestion
    if (_detour_memory_heap != NULL && _detour_memory_heap != NtGetProcessHeap())
```
`_detour_memory_heap` will fallback to process heap if `RtlCreateHeap` failed.

Reply:
Good catch, done.